lib/libutee/tee_api_objects.c

```diff
@@ -571,7 +574,10 @@ TEE_Result TEE_StartPersistentObjectEnumerator(TEE_ObjectEnumHandle
 
 	res = utee_storage_start_enum(objectEnumerator, storageID);
 
-	if (res != TEE_SUCCESS && res != TEE_ERROR_ITEM_NOT_FOUND)
+	if (res != TEE_SUCCESS &&
```

Comment:
I would prefer to have always the same kind of programming, that is here:

	if (res == TEE_SUCCESS ||
	    res == TEE_ERROR_ITEM_NOT_FOUND ||
	    res == TEE_ERROR_CORRUPT_OBJECT ||
	    res == TEE_ERROR_STORAGE_NOT_AVAILABLE)
		return res;
    TEE_Panic(0);